dask_deltatable/core.py

```diff
@@ -8,14 +8,23 @@
 
 import dask
 import dask.dataframe as dd
-import pyarrow.parquet as pq  # type: ignore[import]
+import pyarrow.parquet as pq
 from dask.base import tokenize
 from dask.dataframe.utils import make_meta
 from dask.delayed import delayed
 from deltalake import DataCatalog, DeltaTable
 from fsspec.core import get_fs_token_paths  # type: ignore[import]
 from pyarrow import dataset as pa_ds
 
+from .types import Filters
+from .utils import get_partition_filters
+
+try:
+    filters_to_expression = pq.filters_to_expression
+except AttributeError:
+    # fallback to older internal method
+    filters_to_expression = pq._filters_to_expression
```

Comment:
nit: with these things it's typically helpful to check the version of the library and switch behavior based on this version. this makes it much easier to get rid of it again. With the generic AttributeError we never know when this is safe to be removed.